src/test/java/net/snowflake/client/jdbc/CustomProxyLatestIT.java

```diff
@@ -511,6 +526,7 @@ public PasswordAuthentication getPasswordAuthentication() {
     _connectionProperties.put("user", user);
     _connectionProperties.put("password", passwd);
     _connectionProperties.put("role", "SYSADMIN");
+    _connectionProperties.put("tracing", "ALL");
```

Comment:
These tests are all @Ignored, so it is okay to have tracing set (usually end up turning it on during manual testing)